README.md

```diff
@@ -1,11 +1,14 @@
 # prodmgr
 
-`prodmgr` is the CLI for Shasta uninstall and downgrade. It works by running
-product-specific install utility container images.
+`prodmgr` is the CLI for Shasta activation or deletion of versions of products. It works by running
+product-specific install utility container images to activate products and a generic deletion container
+to delete them.
 
 ## Assumptions made by prodmgr
 
-For a product to work with `prodmgr`, an install utility image must exist
+A generic image already exists for deleting products with `prodmgr`.
+
+However, for a product to be activated with `prodmgr`, an install utility image must exist
 for that product, specifically for the version of the product being operated
 on.
```

Comment:
This is a little unclear still, IMO. We should be really clear that because the generic utility exists, most products do not need to do anything to get "delete" support and that the product-specific install utility image is just a fallback.

Reply:
Made clearer.